code/modules/events/unknown.dm

```diff
@@ -0,0 +1,47 @@
+/datum/event/spawn_unknown
+	var/key_of_unknown
+
+/datum/event/spawn_unknown/proc/get_unknown(var/end_if_fail = 0)
+	spawn()
+		var/list/candidates = pollCandidates("Do you want to play as an unknown entity?", ROLE_DEMON, 1, 300, 0, 24)
```

Comment:
We're not limited on special role IDs, you should be able to update the roles list with your new antag